abnf/odata-aggregation-abnf.txt

```diff
@@ -79,28 +82,37 @@ primitiveProperty =/ expressionAlias / customAggregate
 apply      = ( "$apply" / "apply" ) EQ applyExpr
 applyExpr  = applyTrafo *( "/" applyTrafo )
 applyTrafo = aggregateTrafo
+           / topcountTrafo

```

Comment:
Why give up on the lexical ordering - maybe add the semantics driving the new ordering as comment?

Reply:
Fixed: now ordered lexicographically